docker/insolard-entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+export IP=`awk 'END{print $1}' /etc/hosts`
```

Comment:
Ditto.

Reply:
ok, will do

Reply:
It's everything about docker container. It's one way to obtain docker public ip of local container for raw docker